### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.j-list-limit-label)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#62]
#### Issue
![image](https://user-images.githubusercontent.com/1296369/69328285-e3f5c400-0c46-11ea-9e94-f27da5dffe08.png)

`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Templates: Styles (Site) - kh - Administration](http://localhost/kh/administrator/index.php?option=com_templates&view=styles)
#### Element path
.j-list-limit-label
#### Snippet
`<div class="j-list-limit-label">
				Items per page: 			</div>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.4 (foreground color: #5e727c, background color: #edf0f5, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97
